src/ppom-controller.ts

```diff
@@ -1031,6 +1031,10 @@ export class PPOMController extends BaseController<
       );
     }
 
+    if (this.#chainId !== ETHEREUM_CHAIN_ID) {
+      return undefined;
+    }
```

Comment:
the title of the PR is "Intiialise PPOM only if user is on mainnet". why are we initializing from line 1006 and checking the files before we get to this check? 

cc: @jpuri

Reply:
@jpuri can correct me, but the `initialisePPOM` is actually just loading the wasm code. `this.#ppomProvider.PPOM.new(...)` is where we create a new instance of ppom, and we only want to do that for Ethereum mainnet (for now).

(But yeah I agree that naming seems to be a bit confusing.. it might be worth reviewing it in the future)